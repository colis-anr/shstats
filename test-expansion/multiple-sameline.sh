#!/bin/sh

x=2 y=3 z=4 : /bin/true
echo $x $y
