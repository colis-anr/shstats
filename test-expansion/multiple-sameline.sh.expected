[
  [
    "CompleteCommand_CList",
    [
      "CList_AndOr",
      [
        "AndOr_Pipeline",
        [
          "Pipeline_PipeSequence",
          [
            "PipeSequence_Command",
            [
              "Command_SimpleCommand",
              [
                "SimpleCommand_CmdPrefix_CmdWord_CmdSuffix",
                [
                  "CmdPrefix_CmdPrefix_AssignmentWord",
                  [
                    "CmdPrefix_CmdPrefix_AssignmentWord",
                    [
                      "CmdPrefix_AssignmentWord",
                      [ [ "Name", "x" ], [ "Word", "2", [] ] ]
                    ],
                    [ [ "Name", "y" ], [ "Word", "3", [] ] ]
                  ],
                  [ [ "Name", "z" ], [ "Word", "4", [] ] ]
                ],
                [ "CmdWord_Word", [ "Word", ":", [] ] ],
                [ "CmdSuffix_Word", [ "Word", "/bin/true", [] ] ]
              ]
            ]
          ]
        ]
      ]
    ]
  ],
  [
    "CompleteCommand_CList",
    [
      "CList_AndOr",
      [
        "AndOr_Pipeline",
        [
          "Pipeline_PipeSequence",
          [
            "PipeSequence_Command",
            [
              "Command_SimpleCommand",
              [
                "SimpleCommand_CmdName_CmdSuffix",
                [
                  "CmdName_Word",
                  [ "Word", "echo", [ [ "WordName", "echo" ] ] ]
                ],
                [
                  "CmdSuffix_CmdSuffix_Word",
                  [ "CmdSuffix_Word", [ "Word", "2", [] ] ],
                  [ "Word", "3", [] ]
                ]
              ]
            ]
          ]
        ]
      ]
    ]
  ]
]